next/13/replace-next-head-v2/index.ts

```diff
@@ -894,9 +894,10 @@ const initTsMorphProject = async (
 
 	const allFilePaths = await unifiedFileSystem.getFilePaths(
 		rootPath,
-		['**/*.{jsx,tsx}'],
+		['**/*.{jsx,tsx,ts,js}'],
```

Comment:
should `.mdx`, `.cjs`, `.ejs` also be taken into account?

Reply:
i will add mdx support later